### PR TITLE
Fix: iOS 15 group detail view controller has clear navi bar background

### PR DESCRIPTION
**Pull request by billypchan**
### Issues

On iOS 15, group detail view has a clear background instead of white color

### Causes

iOS 15 changes the default navigation bar behaviour, related discussion
https://developer.apple.com/forums/thread/682420

setting the Navigation Bar's background image is no longer works

### Solutions

Set the background color of navigation view controller for the affected view controllers. I also tried set the UI appearance of navigator bar but it overrides the config of navigation bar(font, tint color...).

Regenerate all snapshot tests and clean up snapshots for removed tests.

### discussion 
We should think about refactor the navigation bar design for iOS 15?

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoRootViewController.swift [hunk 1]
```diff
@@ -56,7 +56,7 @@ final class CallInfoRootViewController: UIViewController, UINavigationController
          selfUser: UserType) {
         self.configuration = configuration
         contentController = CallInfoViewController(configuration: configuration, selfUser: selfUser)
-        contentNavigationController = contentController.wrapInNavigationController()
+        contentNavigationController = contentController.wrapInNavigationController(setBackgroundColor: false)
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Calling/CallInfoViewController/CallInfoRootViewController.swift:59]:
```suggestion
        contentNavigationController = contentController.wrapInNavigationController()
```

**Comment by typfel** [comment]
@billypchan David is not working today.

**Comment by typfel** [comment]
Why were snapshots deleted?

**Comment by billypchan** [comment]
@typfel I rerun the tests and cleanup the old snapshots.

**Review by KaterinaWire — APPROVED** [review]

**Review by typfel — APPROVED** [review]

**Pull request merged**